omarchy-plugin/tests/test_today.py

```
@@ -0,0 +1,102 @@
+            ({"completed": 2, "goal": 0, "date": "2026-09-04", "localToday": "2026-09-04"}, "2 today"),
+        )
+        helper = os.path.join(os.path.dirname(__file__), "..", "widget", "today.js")
+        script = (
+            "const fs = require('fs');"
+            "const helper = require(process.argv[1]);"
+            "const input = JSON.parse(fs.readFileSync(0, 'utf8'));"
+            "process.stdout.write(helper.formatTodayLine(input.completed, input.goal, input.date, input.localToday));"
+        )
+        for values, expected in cases:
+            with self.subTest(values=values):
+                result = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)